patches/minecraft/net/minecraft/entity/ai/EntityAIMate.java.patch

```diff
@@ -0,0 +1,30 @@
+--- ../src-base/minecraft/net/minecraft/entity/ai/EntityAIMate.java
++++ ../src-work/minecraft/net/minecraft/entity/ai/EntityAIMate.java
+@@ -95,6 +95,16 @@
+                 entityplayer = this.field_75391_e.func_146083_cb();
+             }
+ 
++            this.field_75390_d.func_70873_a(6000);
```

Comment:
rather then move this hunk around without any explination, why don't you just fire the event before the if (entityageable != null).
Or! you could gate the if (entityplayer != null) with a if (entityageable != null) and add the return down lower to make the patch smaller and easier to read.

Or, at the very least, give an explination of what this hunk of code is.